### DISCUSSION TITLE
Инструкция по верификации аналитики

### DISCUSSION THREAD

**Discussion by asmazovec** [vpo-tusur/todo-api#24] [QA и Тестирование]
> Пишите сюда в комментарии, обсуждайте, какие методы верификации нашли и использовали для проверки аналитики.

## Подготовка
0) Ознакомься с тем, что есть в дискуссии по онбордингу и тестированию, там описаны основные моменты по организации командной работы:

    - онбординг - [https://github.com/vpo-tusur/todo-api/discussions/11](https://vk.com/away.php?to=https%3A%2F%2Fgithub.com%2Fvpo-tusur%2Ftodo-api%2Fdiscussions%2F11&utf=1)
    - доки по тестированию - https://github.com/orgs/vpo-tusur/discussions/categories/qa-%D0%B8-%D1%82%D0%B5%D1%81%D1%82%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5

## Верификация
1) Переходим на доску QA ([https://github.com/orgs/vpo-tusur/projects/2/views/7](https://vk.com/away.php?to=https%3A%2F%2Fgithub.com%2Forgs%2Fvpo-tusur%2Fprojects%2F2%2Fviews%2F7&utf=1))
2) Выбираем задачу из колонки "Верификация аналитики" и делаем себя в этой задаче исполнителем (заходишь в карточку задачи и справа вверху выбираешь assingnes, находишь там себя). Должно получиться 2 автора - ты и тот, кто завел задачу.
<img src="https://github.com/vpo-tusur/todo-api/assets/60359897/002f8dc9-9ba4-48a9-83b6-e2d621b9bc97" width="500" />

3) Проверяем задачу методами верификации
4) ...
    а) Если все окей переносишь задачу в колонку "Бэклог"
    б) Если есть ошибки пишем по ним комментарий внутрь задачи и переводим задачу обратно в колонку "Аналитика"

## Финализация

5) Последним шагом нужно **обязательно** поделиться полученным опытом верификации с коллегами в комментариях к этой дискуссии:
    - как проводилась верификация аналитики,
    - на что опирался, на что обращал особое внимание,
    - какими подходами пользовался,
    - какие выводы можно сделать
    - какие рекомендации можно дать коллегам,
    - ... любые твои мысли, идеи, замечания, которые могут помочь другим в такой же задаче

**Comment by Dschaymaa** [comment]
Если знаете как делать, покажите мне пример или ссылку на пример

**Reply by asmazovec** [reply]
Уже есть несколько хороших примеров работы над задачей, можешь изучить задачи и комментарии к ним:
1) https://github.com/vpo-tusur/todo-api/issues/18
2) https://github.com/vpo-tusur/todo-api/issues/23

Задачи двигаются по колонкам в процессе работы, текст задачи тоже меняется чтобы исправлять замечания. Это тоже необходимо учитывать при изучении.
